api/v1alpha1/proxmoxmachine_types.go

```diff
@@ -202,6 +207,14 @@ type VirtualMachineCloneSpec struct {
 	Target *string `json:"target,omitempty"`
 }
 
+// TemplateSelector defines tags for looking up images.
+type TemplateSelector struct {
+	// Specifies all tags to look for, when looking up the VM template.
+	//
+	// +kubebuilder:validation:MinItems=1
```

Comment:
Okay, we need to do that in the webhook (do we need to do that? Not really, but it would've been nice to just not eat the yaml here).
Basically, this used to be possible, but now is no longer allowed.
The reasoning seems to be that algorithmic complexity is deemed O(n²) (every element is going to be compared to every element). This is clearly stupid because an actual uniqueness test is just a sort, which is O(n*log(n)), which is much better than what Kubernetes is complaining about ...